.github/ISSUE_TEMPLATE/github-sponsors-profile-review.yml

```diff
@@ -0,0 +1,16 @@
+name: GitHub Sponsors Profile Review
+description: Please review my GitHub Sponsors profile on a livestream
+title: "[REVIEW] <profile name here>"
+labels: [gh sponsor review, live stream review]
+body:
+  - type: input
+    id: link
+    attributes:
+      label: GitHub Sponsor profile link
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+        You can also join our Discord community [here](http://discord.eddiehub.org)
+        Feel free to check out other cool repositories of the EddieHub Community [here](https://github.com/EddieHubCommunity)
```

Comment:
Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
        Feel free to check out other cool repositories of the EddieHub Community [here](https://github.com/EddieHubCommunity)

```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

Reply:
Good spot 👍